### DISCUSSION TITLE
WebUI 2 Status

### DISCUSSION THREAD

**Discussion by kristenmills** [Flexget/webui#42] [Announcements]
This will pretty much serve as a places to see the current status of webui and what's being worked on

## Github Projects
[Configurable Homepage](https://github.com/Flexget/webui/projects/5)
[Standalone webui](https://github.com/Flexget/webui/projects/4)
[Config Editor](https://github.com/Flexget/webui/iprojects/3) 
[Chores/QA/Bugs/etc](https://github.com/Flexget/webui/projects/2)

## Supported Features
 **Section** | **Functionality** |
| --- | --- |
| Log | <li>View real-time logs</li><li>Search in the logs (by task, item, keyword, ...)</li>|
| History | <li>View latest accepted history</li><li>Search by task</li> |
| Pending List | <li>Approve and Reject entries in a list</li><li>Create/remove a new Pending List</li><li>Inject entries into a task</li><li>Add new entry to a pending list</li><li> Bulk pending list operations</li>|
| Entry List | <li>Create/remove a new Entry List</li><li>Inject entries into a task</li><li>Add new entry to an Entry list</li><li> Bulk Entry list operations</li>|
| Movie List | <li>Create/remove a new Movie List</li><li>Inject movies into a task</li><li>Add new entry to an Movie list</li><li> Bulk Movie list operations</li>|
| Series| <li>View All Series episodes and releases</li><li>Add new series</li><li>Remove releases/reset</li><li>Remove episodes</li><li>Remove/Reset all releases for an episode</li>|
| Tasks | <li>View Latest Tasks executions</li><li>View all executions for a task</li>|
| Config Editor| <li>viewing and editing config and variable yaml in the browser</li>|
| Miscellaneous | <li>Shutdown Flexget</li><li>Reload config</li><li>DB Operations (vacuum, cleanup, reset plugin)</li> |

**Comment by paranoidi** [comment]
Awesome work! In order to benefit casual and lazy (like me) users the end game should be to generate and update configuration file from UI. Check my old mockups from: https://flexget.mybalsamiq.com/projects/webui/Series%20-%20Set%20up

**Comment by kristenmills** [comment]
@paranoidi Yeah absolutely agreed.  That's the follow up to the stuff in this milestone along with how to properly support custom web plugins and configurable views and etc as well.
